docusaurus/docs-internals/editor-controls/data-controls/codeMirror.md

```diff
@@ -0,0 +1,253 @@
+---
+toc_max_heading_level: 4
+---
+# Code Mirror
+
+Brizy's code control features a code editor with a textarea interface.
+
+Example of code mirror control:
+
+![Code Mirror](/img/data-controls/codeMirror.png)
+
+### Parameters
+
+| Name               | Type        |    Default    | Description                                                                              |
+|:-------------------|:------------|:-------------:|:-----------------------------------------------------------------------------------------|
+| `id`               | `string`    |       -       | The identifier of the key where the `codeMirror` will save your data                     |
+| `type`             | `string`    |       -       | Type should be `"codeMirror"` to use this control                                        |
+| `placeholder?`     | `string`    |      -        | The placeholder text displayed in the input field.                                       |
+| `label?`           | `string`    |       -       | The label displayed on the left side of the control                                      |
+| `position?`        | `number`    |       -       | The position of the control in toolbar                                                   |
+| `devices?`         | `"all"` \| `"desktop"` \| `"responsive"`  | `"all"` | Define the devices where the control will be rendered. `"all"` renders the control on all devices. `"desktop"` renders the control only on desktop devices. `"responsive"` renders the control on both tablet and mobile devices. |
+| `disabled?`        | `boolean`   |    `false`    | Configure the condition under which the control is disabled or enabled.                  |
+| `helper?.content`  | `string`   |    -    | If provided, an icon is displayed next to the label. When hovering over this icon, a tooltip with additional information appears.                                                                                                 |
+| `helper?.position` | `"top-start"` \| `"top"` \| `"top-end"` \| `"right-start"` \| `"right"` \| `"right-end"` \| `"bottom-end"` \| `"bottom"` \| `"bottom-start"` \| `"left-end"` \| `"left"` \| `"left-start"` | `"top"` | Specifies the position of the tooltip relative to the helper icon.                                                                                                                                                                |
+| `config?.size`     | `Size`      |  `"medium"`   | The size of textarea on the left side of the control     <br/> <br/> `Size = "short" \| "medium" \| "large" \| "auto"`  |                 
```

Comment:
about size:

write like in Slider:
![image](https://github.com/EasyBrizy/Brizy-Local-Editor/assets/52622998/9ff1a185-ce92-4129-9e55-4f36a11cf6be)